component/doc/README.md

```diff
@@ -0,0 +1,346 @@
+
+* [Step 1 - Configure the LinkedIn App](#step-1---configure-the-linkedin-app)
+* [Step 2 - Deploy the Pickup Dispatch Sample Web App](#step-2---deploy-the-pickup-dispatch-sample-web-app)
+* [Step 3 - Configure the Identity Provider (IdP)](#step-3---configure-the-identity-provider-idp)
+* [Step 4 - Configure the Service Provider](#step-4---configure-the-service-provider)
+* [Step 5 - Configure Claims](#step-5---configure-claims)
+* [Step 6 - Configure Requested Claims for Pickup Dispatch Sample Web App](#step-6---configure-requested-claims-for-pickup-dispatch-sample-web-app)
+* [Step 7 - Test the Sample Application](#step-7---test-the-sample-application)
+
+##Step 1 - Configure the LinkedIn App
+
+1. Place the authenticator.jar file into the 
+`<IS_HOME>/repository/components/dropins` directory. 
+You can download the .jar file 
+(`org.wso2.carbon.extension.identity.authenticator.linkedin.connector-x.x.x`) 
+from the [WSO2 Store](https://store.wso2.com/store/assets/isconnector/list?q=%22_default%22%3A%22LinkedIn%22). 
+Next, start or restart the WSO2 Identity Server. 
+    > :information_source: If you want to upgrade the LinkedIn (.jar) in your existing Identity Server pack, 
+please refer [upgrade instructions](https://docs.wso2.com/display/ISCONNECTORS/Upgrading+an+Authenticator).
+ 
+ 2. Create a new application as described in below.
+ 
+    a. Navigate to the following URL: `https://www.linkedin.com/developer/apps/new`
+    
+    b. Enter the required details.
+    
+        * App Name and Company details. The company should have a linkedin page.
+        * Upload an image that you wish to use at the company logo.
+        * Select the checkbox to agree to the LinkedIn terms and conditions.
+    c. Click on **Create App**. You will redirect to a page with **Settings** for the created application. 
+    
+    ![linkedin-app-create-page](img/linkedin-app-create-page.png)
+    
+    ![linkedin-app-settings-page](img/linkedin-app-settings-page.png)
+    
+    d. Navigate to the **Auth** tab which you can see the **Client ID** and
+     **Client Secret**.
+     
+    e. Enter the **Redirect URL** under the **OAuth 2.0 settings** as the following format and click on **Update
+    **. `https://{hostname}:{port}/commonauth`. The default redirect URL in WSO2 Identity Server is `https
+    ://localhost:9443/commonauth`.
+    
+    ![linkedin-app-auth-page](img/linkedin-app-auth-page.png)
+    
+    f. Copy the **Client ID** and **Client Secret** from the resulting page.
+    
+##Step 2 - Deploy the Pickup Dispatch Sample Web App
+
+The next step is to deploy the Pickup Dispatch sample web app in order to use it in this scenario.
+
+To configure this, see [saml2-web-app-pickup-dispatch 
+webapp](https://is.docs.wso2.com/en/5.9.0/learn/deploying-the-sample-app/#deploying-the-saml2-web-app-pickup-dispatch-webapp).
+
+> :information_source: Please avoid **Configuring the service provider** section in the above reference because we will
```

Comment:
Instead of saying this we can let them follow that section and later instruct the user to "Edit the service provider you created in step2 and do the following configurations"